tests/test_xdsl_base.py

```diff
@@ -1028,37 +1042,20 @@ def test_tan(self, deg, exp):
         assert np.isclose(norm(u), exp, rtol=1e-4)
 
 
-class TestOperatorUnsupported(object):
+def test_forward_assignment():
+    # simple forward assignment
 
-    @pytest.mark.xfail(reason="stencil.return operation does not verify for i64")
-    def test_forward_assignment(self):
-        # simple forward assignment
-
-        grid = Grid(shape=(4, 4))
-        u = TimeFunction(name="u", grid=grid, space_order=2)
-        u.data[:, :, :] = 0
-
-        eq0 = Eq(u.forward, 1)
-
-        op = Operator([eq0], opt='xdsl')
-
-        op.apply(time_M=1)
-
-        assert np.isclose(norm(u), 5.6584, rtol=0.001)
-
-    @pytest.mark.xfail(reason="stencil.return operation does not verify for i64")
-    def test_function(self):
-        grid = Grid(shape=(5, 5))
-        x, y = grid.dimensions
+    grid = Grid(shape=(4, 4))
+    u = TimeFunction(name="u", grid=grid, space_order=2)
+    u.data[:, :, :] = 0
 
-        f = Function(name="f", grid=grid)
+    eq0 = Eq(u.forward, 1)
 
-        eqns = [Eq(f, 2)]
+    op = Operator([eq0], opt='xdsl')
 
-        op = Operator(eqns, opt='xdsl')
-        op.apply()
+    op.apply(time_M=1)
 
-        assert np.all(f.data == 4)
```

Comment:
I would also keep this, just for nitpicking

Reply:
Confusing diff: This is kept and moved to function_V!

Though, I changed it to assert 2 instead of 4, as the input equation is `f = 2` :stuck_out_tongue_winking_eye: